src/components/DefinitionList/index.ts

```diff
@@ -0,0 +1,2 @@
+export {DefinitionList} from './DefinitionList';
```

Comment:
Let's reexport `DefinitionListItem` component as well. There might be issues with dot notation in Next.js applications